src/prActions.ts

```diff
@@ -0,0 +1,47 @@
+import { addComment, needsUpdate, removeLabel, updatePr } from "./github.ts";
+import { debounce } from "https://deno.land/std@0.189.0/async/mod.ts";
+
+const execute = async (
+  label: string,
+  pr: { number: number; user: { login: string } },
+) => {
+  switch (label) {
+    case "giteabot/update-branch": {
+      await updateBranch(pr);
+      await removeLabel(pr.number, "giteabot/update-branch");
+    }
+  }
```

Comment:
```suggestion
  if (label === "giteabot/update-branch") {
    await updateBranch(pr);
    await removeLabel(pr.number, "giteabot/update-branch");
  }
```

More of a personal preference, but I almost never use `switch` in Javascript. The only place I use it is in performance-critical parts as last I checked it is a bit faster then `if` branches.

Reply:
I don't really have a preference, fine with me